src/utils.ts

```diff
@@ -68,20 +69,20 @@ export function createSignInputsFromImportableEd25519Key(key: TImportableEd25519
     throw new Error('No verification method type provided')
 }
 
-export function createKeyPairRaw(): KeyPair {
-    return generateKeyPair()
+export function createKeyPairRaw(seed?: string): KeyPair {
+    return seed ? generateKeyPairFromSeed(Buffer.from(seed)) : generateKeyPair()
 }
 
-export function createKeyPairBase64(): IKeyPair {
-    const keyPair = generateKeyPair()
+export function createKeyPairBase64(seed?: string): IKeyPair {
+    const keyPair = seed ? generateKeyPairFromSeed(Buffer.from(seed)) : generateKeyPair()
     return {
         publicKey: toString(keyPair.publicKey, 'base64'),
         privateKey: toString(keyPair.secretKey, 'base64'),
     }
 }
 
-export function createKeyPairHex(): IKeyPair {
-    const keyPair = generateKeyPair()
+export function createKeyPairHex(seed?: string): IKeyPair {
+    const keyPair = seed ? generateKeyPairFromSeed(Buffer.from(seed)) : generateKeyPair()
```

Comment:
Switch to `uint8arrays` please.
I'd suggest to do a micro-function that handles the optional `seed` variable or re-use `createKeyPairRaw` wherever this condition is needed.